doc/articles/dummies1.md

```diff
@@ -0,0 +1,138 @@
+# How to migrate existing UWP code to Uno
+
+There are two separate paths for using an existing UWP codebase on top of uno:
+- An existing UWP application
+- An existing UWP library
+
+For migrating UWP library, see [this article](howto-migrate-existing-code.md).
+
+# How to migrate UWP app to Uno?
+# Step by step guide.
+
+This guide assumes that you have some UWP app, and want to convert this app to be multiplatform.
+
+ Maybe your programmers experience doesn't include Git, Android, iOS, etc. Maybe you know only UWP. And maybe even your apps are written in VB... But still, you can not only use Uno, but also contribute to it. Please, read this short guide, guide for beginners (or dummies...).
+
+
+In short:
+* convert project into Uno,
+* find if something you use is not implemented in Uno,
+* if yes, you have two options:
+	* try to find workaround for this, or
+	* extend Uno
+* try to run converted app, and test it
+
+
+
+## adding new project
+
+First, open your app solution in VStudio. Then using Solution Explorer window:
+* add new project (right click on Solution, Add, New Project, Visual C#, Uno Plaform, Cross-Plaform App (Uno Platform), using name as e.g. your previous project (APPNAME) with "\_Uno" suffix. In effect, APPNAME_Uno folder would be created, and inside it - folders as APPNAME_Uno.Shared, APPNAME_Uno.Droid, etc. - one project per one target platform.
```

Comment:
The disadvantage of this relative to the protocol in the existing doc is that it will mess up the default namespaces and assembly names (MyApp -> MyApp_Uno). 

I would prefer it if you took the existing protocol and describe it in the level of detail you have here. I know it's not very clear, let me know if it still doesn't make sense now you have more experience with Uno (and Visual Studio).

Reply:
In existing app, all files have namespace defined as "MyApp". Porting app is done (should be...) by copying old files to new project - source, manifest, etc.
But Uno project cannot be created with same name as existing app, at least in same Solution. From Solution Explorer, old MyApp project can be renamed to e.g. MyApp_old, but file system directory name would not be renamed, and VStudio will fail to create new MyApp project (Uno version).

Reply:
Yes, you need to create a separate solution in a separate folder (that's one of the parts that's unclear). Then copy the Uno head projects to your original folder and include them in the original solution.

Reply:
Such change requires bigger rework of this guide, as this guide is written to be separate project, but same solution. 
Can it be left in this form (same solution), or should I rework it to new solution?

Reply:
(and all my (four) projects I have in same solution :) )